PowerEditor/installer/nppSetup.nsi

```diff
@@ -150,6 +150,16 @@ Function .onInit
 	;
 	; --- PATCH END ---
 
+	; handle the possible Silent Mode (/S) & already running Notepad++ (without this an incorrect partial installation is possible)
+	IfSilent 0 notInSilentMode
+	System::Call 'kernel32::OpenMutex(i 0x100000, b 0, t "nppInstance") i .R0'
+	IntCmp $R0 0 nppNotRunning
+	System::Call 'kernel32::CloseHandle(i $R0)' ; a Notepad++ instance is running, tidy-up the opened mutex handle only

```

Comment:
In the case of `nppNotRunning`, is there no mutex handle ?

Reply:
Yes. 
In such case the OpenMutex returns NULL (and possible GetLastError() then shows ERROR_FILE_NOT_FOUND).